### PR TITLE
feat: show specific warning messages if the relevant token is not the one in hand

### PR DESCRIPTION
**Pull request by Nisha1306**
Solves: https://github.com/AtB-AS/kundevendt/issues/220

Scenarios covered:

Case 1: Info on the ticket itself about errors (internet lack ex.)
Case 2: Info on the ticket itself if not token
Case 3: Info on the ticket itself when tokens are missing
Case 4: Info that the t: card is used as a travel document inside the ticket itself
Case 5: Info about mobile token is used as a travel document inside the ticket itself

Eg Case 5:

Screenshot_20220629-154134

### CODE REVIEW

File: src/screens/Ticketing/Ticket/TicketInfo.tsx [hunk 1]
```diff
@@ -96,6 +103,49 @@ export const TicketInfoView = (props: TicketInfoViewProps) => {
...
+      <ThemeText type="body__secondary">
+        {t(TicketTexts.warning.unableToRetrieveToken)}
+      </ThemeText>
+    );
+  if (remoteTokens && remoteTokens.length === 0)
+    return (
+      <ThemeText type="body__secondary">
+        {t(TicketTexts.warning.noTokenFound)}
+      </ThemeText>
+    );
```
Comment (gorandalum) [inline src/screens/Ticketing/Ticket/TicketInfo.tsx:124]: An empty `remoteTokens?` list will mae `isError` `true`, so this clause is redundant. Just showing the `unableToRetrieveToken` is good enough, as something has gone wrong.

**Review by gorandalum — APPROVED** [review]
Nice! Just a note about some code that can be removed.

File: src/screens/Ticketing/Ticket/TicketInfo.tsx (hunk 1 above)
Comment (Nisha1306) [reply src/screens/Ticketing/Ticket/TicketInfo.tsx:124]: Ohh Oka. I will get rid of this. It was in one of the cases mentioned in the issue, so I thought its a valid scenario :)

File: src/screens/Ticketing/Ticket/TicketInfo.tsx [hunk 2]
```diff
@@ -169,12 +204,10 @@ const TicketInfoTexts = (props: TicketInfoViewProps) => {
           {tariffZoneSummary}
         </ThemeText>
       )}
-      {showTravelCardActiveWarning && (
-        <View style={styles.notInspectableWarning}>
-          <ThemeIcon svg={Warning} style={styles.notInspectableWarningIcon} />
-          <ThemeText isMarkdown={true}>
-            {t(TicketTexts.ticketInfo.travelcardIsActive)}
-          </ThemeText>
+      {warning && (
```
Comment (Nisha1306) [inline src/screens/Ticketing/Ticket/TicketInfo.tsx:207]: @Gøran Dalum  I haven't put this under any flag/condition like "show only for new tokens" or something. Is there anything like that, which should to be considered?

Comment (gorandalum) [reply src/screens/Ticketing/Ticket/TicketInfo.tsx:207]: Not that I can think of now

**Pull request merged**